src/main/java/io/strimzi/kafka/bridge/tracing/OpenTelemetryHandle.java

```diff
@@ -65,11 +68,26 @@ public String serviceName(BridgeConfig config) {
     @Override
     public void initialize() {
         System.setProperty("otel.metrics.exporter", "none"); // disable metrics
+        final String configuredOtlpProtocol = resolveConfiguredOtelProtocol();
+        if (configuredOtlpProtocol == null || configuredOtlpProtocol.equalsIgnoreCase("grpc")) {
+            LOGGER.warn("The HTTP-Kafka bridge is deprecating OTLP GRPC support in favour of plain HTTP transports. Please switch to http/protobuf and update collector addresses to match");
+        }
         // TODO: to remove when Vert.x won't be used anymore and the ThreadLocalContextStorage could be used again
         System.setProperty("io.opentelemetry.context.contextStorageProvider", "io.strimzi.kafka.bridge.tracing.BridgeContextStorageProvider");
         AutoConfiguredOpenTelemetrySdk.initialize();
     }
 
+    private static String resolveConfiguredOtelProtocol() {
+        final Map<String, String> envVars = System.getenv();
+        final String tracesProtocol = System.getProperty("otel.exporter.otlp.traces.protocol", envVars.get("OTEL_EXPORTER_OTLP_TRACES_PROTOCOL"));
+        //The traces protocol is more specific than the general protocol, so it should take prescience
+        if (tracesProtocol != null) {
+            return tracesProtocol;
+        } else {
+            return System.getProperty("otel.exporter.otlp.protocol", envVars.get("OTEL_EXPORTER_OTLP_PROTOCOL"));
+        }
+    }
```

Comment:
Please update the docs to explain the difference between those two and their expected and default values.